rules/windows/process_creation/win_susp_runonce_execution.yml

```diff
@@ -0,0 +1,29 @@
+title: Run Once Task Execution as Configured in Registry
+id: 198effb6-6c98-4d0c-9ea3-451fa143c45c
+description: This rule detects the execution of Run Once task as configured in the registry
+author: Avneet Singh '@v3t0_', oscd.community
```

Comment:
could you please apply [this](https://github.com/Neo23x0/sigma/pull/1210/files#r506965174) recommendation here as well?